yang-module-versioning/draft-ietf-netmod-yang-module-versioning.xml

```diff
@@ -803,145 +793,136 @@ module ietf-yang-revisions {
 
+       Adding a revision label scheme is a backwards-compatible
+       version change.  Changing a revision label scheme is a
+       non-backwards-compatible version change, unless the new
+       revision label scheme is backwards-compatible with the
+       replaced revision label scheme.  Removing a revision label
+       scheme is a non-backwards-compatible version change.";
     reference
       "XXXX: Updated YANG Module Revision Handling;
-      Section 3.3.1, Revision label scheme extension statement";
+       Section 3.3.1, Revision label scheme extension statement";
   }
 
   extension recommended-min {
     argument revision-date-or-label;
     description
       "Recommends the revision of the module that may be imported to
-      one that matches or is derived from the specified
-      revision-date or revision label.
+       one that matches or is derived from the specified
+       revision-date or revision label.
 
-      The argument value MUST conform to the
-      'revision-date-or-label' defined type.
+       The argument value MUST conform to the
+       'revision-date-or-label' defined type.
 
-      The statement MUST only be a substatement of the import
-      statement.  Zero, one or more 'recommended-min' statements
-      per parent statement are allowed.  No substatements for this
-      extension have been standardized.
+       The statement MUST only be a substatement of the import
+       statement.  Zero, one or more 'recommended-min' statements per
+       parent statement are allowed.  No substatements for this
+       extension have been standardized.
 
-      If specified multiple times, then any module revision that
-      satisfies at least one of the 'recommended-min' statements
-      is an acceptable recommended revision for import.
+       If specified multiple times, then any module revision that
+       satisfies at least one of the 'recommended-min' statements is
+       an acceptable recommended revision for import.
 
-      A particular revision of an imported module adheres to an
-      import's 'recommended-min' extension statement if the
-      imported module's revision history contains a revision
-      statement with a matching revision date or revision label.
-      
-      Adding, removing or updating a 'recommended-min'
-      statement to an import is a backwards-compatible change.
-      ";
+       A particular revision of an imported module adheres to an
+       import's 'recommended-min' extension statement if the imported
+       module's revision history contains a revision statement with a
+       matching revision date or revision label.
 
+       Adding, removing or updating a 'recommended-min' statement to
+       an import is a backwards-compatible change.";
     reference
-      "XXXX: Updated YANG Module Revision Handling;
-      Section 4, Recommending a minimum revision for module imports";
+      "XXXX: Updated YANG Module Revision Handling; Section 4,
```

Comment:
Why move "Section 4", is this for line width?

Reply:
Yes, IIRC.